scripts/publish-vscode-extension.ts

```diff
@@ -34,6 +34,7 @@ async function updateExtensionVersion() {
 }
 
 async function packageAndPublish() {
+	await exec('yarn', ['build'])
```

Comment:
this will build everything in the repo, you probably don't want all that so you can probably get away with just this?
```suggestion
	await exec('yarn', ['build-package'])
```